ep/build-velox/src/build_velox.sh

```diff
@@ -122,6 +122,8 @@ function compile {
     make $COMPILE_TYPE EXTRA_CMAKE_FLAGS="${COMPILE_OPTION}"
   elif [ $ARCH == 'arm64' ]; then
     CPU_TARGET="arm64" make $COMPILE_TYPE EXTRA_CMAKE_FLAGS="${COMPILE_OPTION}"
+  elif [ $ARCH == 'aarch64' ]; then
```

Comment:
Nit:
It would be better to unify the handling for arm64/aarch64, i.e., use one `elif` block and let `CPU_TARGET = $ARCH`. Thanks!

Reply:
updated